README.md

```diff
@@ -1,72 +1,92 @@
+# Catalog of my things
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
-
-> Description of the project.
-
+> In this project, we created console app that helps users to keep a record of different types of things we won: books, music albums, movies, and games. Everything is stored on a JSON file and can be retrieved even after the user exit from the application.
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- **Ruby**
 
-## Live Demo (if available)
+## Setup
 
-[Live Demo Link](https://livedemo.com)
+- Install Ruby in local envinronment.
+- Run program(irb) using the command line
 
+## Prerequisites
+- Understand Ruby basic system and linux
+- No linter errors
 
+## Video link
+- [Video Presentation](https://drive.google.com/file/d/1M8U-6YQBTusXeqdgKs2aOFI9UGJoW1qX/view?usp=sharing)
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+Clone the project, make sure you have ruby installed and you are good to go:
 
+```
+   git clone https://github.com/GedeonTS/Ruby-capstone-project.git
+   ``` 
 
-To get a local copy up and running follow these simple example steps.
+To view the project directory
 
-### Prerequisites
+```
+   cd School-library
+   ```
 
-### Setup
+Install the necessary packages
 
-### Install
+```
+  bundle install
+  
+  gem install rubocop
```

Comment:
- [ ] I couldn't find any `Gemfile` on your project that's why the bundle install command says "Could not locate Gemfile". Please work on it. :+1:

Reply:
@mahbubul14 we have added it, Thanks for pointing that out